src/components/articles/CreateArticle.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+import PropType from 'prop-types';
+import { connect } from 'react-redux';
+import ReactFileReader from 'react-file-reader';
+import { createArticle, addTag, removeTag } from '../../actions/article';
+import Layout from '../layouts/Layout';
+import Loading from '../layouts/Loading';
+
+export class CreateArticle extends Component {
+  constructor() {
+    super();
+    this.state = {
+      title: '',
+      image: '',
+      tag: '',
+      loading: false,
+      open: false
+    };
+    this.BodyContent = React.createRef();
+  }
+
```

Comment:
You do not need a constructor when using arrow functions.  Just declare the state.